app/main.py

```diff
@@ -1,8 +1,63 @@
 class Car:
-    # write your code here
-    pass
-
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        total_income = 0  # Renamed from income to total_income
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                total_income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(total_income, 1)
+
+    def calculate_washing_price(self, car):
+        # Calculate cost based on the given formula
+        return round(car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating / self.distance_from_city_center, 1)
+
+    def wash_single_car(self, car):
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
```

Comment:
The condition `if car.clean_mark < self.clean_power:` in the `wash_single_car` method is redundant because it is already checked in the `serve_cars` method before calling `wash_single_car`. You can remove this condition to simplify the code.